sendgrid/sendgrid.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, **opts):
         self.path = opts.get('path', os.path.abspath(os.path.dirname(__file__)))
         self._apikey = opts.get('apikey', os.environ.get('SENDGRID_API_KEY'))
         # Support v2 api_key naming
-        self._apikey = opts.get('api_key', self._apikey)
+        self._api_key = opts.get('api_key', self._apikey)
         self._api_key = self._apikey

```

Comment:
Yes. You are right. 
It should accept v2 key 'api_key' if passed else default to v3 'apikey' or env 'SENDGRID_API_KEY'.